app/presenters/content_change_presenter.rb

```diff
@@ -29,7 +30,12 @@ def call
   delegate :title, :description, :change_note, :footnote, to: :content_change
 
   def content_url
-    PublicUrlService.url_for(base_path: content_change.base_path)
+    utm_source = content_change.id
+    utm_medium = "email"
+    utm_campaign = "govuk-notifications"
+    utm_content = @frequency
```

Comment:
It'd be a bit more consistent with this class and code base to have frequency behind an attr_reader

Reply:
Done